tests/spread/integration/less/task.yaml

```diff
@@ -0,0 +1,11 @@
+summary: Integration tests for less
+
+execute: |
+  rootfs="$(install-slices less_bins ncurses-base_terminfo)"
+
+  chroot "${rootfs}/" less --version
+  chroot "${rootfs}/" lesskey --version
+  echo "Hello, World" > "${rootfs}/testfile.txt"
+  test "$(chroot "${rootfs}/" less -F -X -R testfile.txt)" = "Hello, World"
+  test "$(chroot foo lessecho -m: a:b )" = "a\:b"
+  
```

Comment:
It appears there is an extra blankline at the end of this file that we can remove?